tests/test.py

```diff
@@ -0,0 +1,193 @@
+#!/usr/bin/env python
+# Copyright 2019 AstroLab Software
+# Author: Abhishek Chauhan
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import unittest
+import os
+import io
+import confluent_kafka
+import json
+import glob
+import time
+import fastavro
+from typing import Iterable
+from fink_client.consumer import AlertConsumer
+
+def read_avro_alerts(data_path: str) -> Iterable[dict]:
```

Comment:
Is there any reason to keep those routines (`read_avro_alerts`, `encode_into_avro` and `get_legal_topic_name`) here? I would rather put those in a module under the `fink_client` package if possible.

Reply:
Note: the move can be done in https://github.com/astrolabsoftware/fink-client/pull/20 if this easier to manage.

Reply:
Thanks @JulienPeloton! You are right, It would be better to put them into a module.
I will do the move in #20 as it would be easier.

Reply:
> I would rather put those in a module under the fink_client package if possible.

Done in #20 commit ab4ea4f.

Reply:
Thanks! Merging this one then.